gridly/src/grid/view.rs

```diff
@@ -461,6 +461,18 @@ where
     }
 }
 
+#[cfg(feature = "array_impls")]
+impl<T: Sized, const N: usize, const M: usize> Grid for [[T; M]; N] {
+    type Item = T;
+
+    #[inline]
+    unsafe fn get_unchecked(&self, location: Location) -> &Self::Item {
+        self[..]
```

Comment:
I believe the `[..]` can be omitted here: https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=9d45ecf72fa660a0e264cf991357f971

Reply:
In this case I used the `[..]` so that the call is resolved to `slice.get_unchecked` rather than the `Grid` impl, I can make it more explicit